apis/nucleus/src/components/Supernova.jsx

```diff
@@ -35,6 +35,9 @@ function Supernova({ sn, snOptions: options, snPlugins: plugins, layout, appLayo
     setSnNode(ref);
   }, []);
 
+  const bgComp = layout && layout.components ? layout.components.find((comp) => comp.key === 'general') : null;
+  const bgImage = bgComp && bgComp.bgImage && bgComp.bgImage.mediaUrl ? bgComp.bgImage.mediaUrl : undefined;
```

Comment:
Should we also support expressionUrl?

Reply:
We could do that, but then the `qStringExpression` would need to be evaluated through the expression editor first, right?

Reply:
In that case, could we just put the evaluated URL and the mediaUrl in the same property, `qStaticContentUrlDef` in the mediaURL object and then render the image based on that?

Reply:
Given this spec we agreed upon earlier:
```
styles: {
  bgColor: {
    color: { ...  } // support for 'none' 
  },
  bgImage: {
    mode: none, media, expression
    position: top-left, top-center, top-right, center-left, center-center. center-right, bottom-left, bottom-center, bottom-right,
    sizing: original, alwaysFit, alwaysFill, stretch, fitWidth, fitHeight
    mediaUrl: { qStaticContentUrlDef...}
    expressionUrl: { qStringExpression... }
  }
}
```
You should be able to get the image URL either from the mediaUrl property or from the expressionUrl property. This should be based of the mode property.